src/displayapp/widgets/StatusIcons.cpp

```diff
@@ -37,6 +43,8 @@ void StatusIcons::Update() {
     batteryIcon.SetBatteryPercentage(batteryPercent);
   }
 
+  lv_obj_set_hidden(alarmIcon, !alarmController.IsEnabled());
```

Comment:
Not an LVGL expert, but I'm pretty sure this should use DirtyValue like the other icons as LVGL doesn't track what's on the screen

Reply:
I see, I haven't much experience with LVGL either. I hope it is better now.